.gitignore

```diff
@@ -1,3 +1,6 @@
 node_modules
 bundle.js
-data
\ No newline at end of file
+data
+.env
+newrelic.js
+newrelic_agent.log
```

Comment:
Small suggestion
newrelic*

use * for wild card

Reply:
cool i like this idea!